newrelic/resource_newrelic_synthetics_monitor.go

```diff
@@ -26,53 +27,51 @@ func resourceNewRelicSyntheticsMonitor() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
-				Description: "The monitor type. Valid values are SIMPLE, BROWSER, SCRIPT_BROWSER, and SCRIPT_API.",
+				Description: "The monitor type. Valid values are SIMPLE AND BROWSER.",
 				ValidateFunc: validation.StringInSlice([]string{
 					"SIMPLE",
 					"BROWSER",
-					"SCRIPT_API",
-					"SCRIPT_BROWSER",
 				}, false),
 			},
 			"name": {
 				Type:        schema.TypeString,
 				Required:    true,
 				Description: "The title of this monitor.",
 			},
-			"frequency": {
-				Type:         schema.TypeInt,
-				Required:     true,
-				ValidateFunc: intInSlice([]int{1, 5, 10, 15, 30, 60, 360, 720, 1440}),
-				Description:  "The interval (in minutes) at which this monitor should run. Valid values are 1, 5, 10, 15, 30, 60, 360, 720, or 1440.",
+			"period": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				Description:  "The interval at which this monitor should run. Valid values are EVERY_MINUTE, EVERY_5_MINUTES, EVERY_10_MINUTES, EVERY_15_MINUTES, EVERY_30_MINUTES, EVERY_HOUR, EVERY_6_HOURS, EVERY_12_HOURS, or EVERY_DAY.",
```

Comment:
I understand why we changed the attribute name, but couldn't we keep the integer values? They are pretty much the same, but this adds additional step for customer to migrate to the new resource.